package.json

```diff
@@ -31,7 +31,6 @@
     "enzyme-adapter-react-16": "^1.1.1",
     "eslint": "^4.19.1",
     "eslint-config-airbnb": "^17.0.0",
-    "eslint-config-postcss": "^2.0.2",
```

Comment:
I removed this one completely because version `3.0.2` has a lot of peer dependencies we don't have and because I do not see us extending this config in the `.eslintrc` file so my guess is it was removed at one point?

https://github.com/postcss/eslint-config-postcss/blob/master/package.json